### ISSUE TITLE
Where tooltips on '?' icons are the exact same as the question, is it necessary?

### ISSUE THREAD

**Issue by pacharanero** [rcpch/rcpch-audit-engine#746]
![image](https://github.com/rcpch/rcpch-audit-engine/assets/2348385/a21b457b-ab94-4972-b3f5-2f8f23fde0bd)

In a lot of the questions, the text in the tooltip is identical to the question being asked, so serves no function.

### Options:

1. Remove them all (where they are the same as the question)

2. Change what is in the tooltips so that is links to some external reference as to why that question is being asked? Eg this could link to more information on KPIs in the documentation?

Views @eatyourpeas @AmaniKrayemRCPCH @nikyraja

**Comment by eatyourpeas** [comment]
The `help_text` field in the django models is only a single string, so the method was subclassed to allow us to have both label text and help text accessible from the same field. The `label` key is meant to be the label in the form, and the `reference` key relates to any documents or standards referenced by that field. So for ECG, for example, the `reference` was meant to reference both the citation in the audit document prescribing the measure, as well as reference any national standards (eg NICE) that were associated with it. Not all measures have associated standards though, and it is tricky to have an empty tooltip, so where there is no reference, the label text has been duplicated. 
I guess we could remove the help icons next to each question if there is no meaningful additional information. This is one probably for the E12 team to guide on.

**Comment by AmaniKrayemRCPCH** [comment]
Hi all, 

Attached is a data dictionary. The last column can be used to populate tooltips. Those that are blank do not need the help icon.
This might be a little out of date, so happy for you to make edits to the tooltip text as needed. 

[Data Dictionary 6.0.docx](https://github.com/rcpch/rcpch-audit-engine/files/13965067/Data.Dictionary.6.0.docx)

**Comment by nikyraja** [comment]
Yes, please can we delete the tooltips where they are exactly the same as the question. The file that Amani shared can eb used to populate these. 

It would be good to add 'Informs KPI X' where applicable for the questions as well. 

Just created issue #949 for a specific update that is in addition to this.
